Sensus.iOS/Info.plist

```diff
@@ -1,176 +1,172 @@
-	<key>NSMotionUsageDescription</key>
-	<string>Sensus uses movement sensors to collect information for studies you enroll in.</string>
-	<key>NSCameraUsageDescription</key>
-	<string>Sensus uses the camera to scan participation barcodes. Sensus will not record images or video.</string>
-	<key>CFBundleShortVersionString</key>
-	<string>15.14.0</string>
-	<key>CFBundleVersion</key>
-	<string>1.1.1</string>
-</dict>
+  <dict>
+    <key>UIFileSharingEnabled</key>
+    <true/>
+    <key>LSSupportsOpeningDocumentsInPlace</key>
+    <true/>
+    <key>CFBundleDisplayName</key>
+    <string>Sensus</string>
+    <key>CFBundleIdentifier</key>
+    <string>edu.virginia.sie.ptl.sensus</string>
+    <key>LSRequiresIPhoneOS</key>
+    <true/>
+    <key>MinimumOSVersion</key>
+    <string>10.0</string>
+    <key>UIDeviceFamily</key>
+    <array>
+      <integer>1</integer>
+    </array>
+    <key>UILaunchStoryboardName</key>
+    <string>LaunchScreen</string>
+    <key>UIMainNibFile</key>
+    <string>LaunchScreen</string>
+    <key>UIRequiredDeviceCapabilities</key>
+    <array>
+      <string>armv7</string>
+    </array>
+    <key>UISupportedInterfaceOrientations</key>
+    <array>
+      <string>UIInterfaceOrientationPortrait</string>
+      <string>UIInterfaceOrientationLandscapeLeft</string>
+      <string>UIInterfaceOrientationLandscapeRight</string>
+    </array>
+    <key>NSLocationAlwaysUsageDescription</key>
+    <string>Sensus uses GPS to collect location information for studies you enroll in.</string>
+    <key>CFBundleURLTypes</key>
+    <array>
+      <dict>
+        <key>CFBundleURLSchemes</key>
+        <array>
+          <string>fb873948892650954</string>
+        </array>
+        <key>CFBundleURLTypes</key>
+        <string>Viewer</string>
+      </dict>
+      <dict>
+        <key>CFBundleURLName</key>
+        <string>edu.virginia.sie.ptl.sensuss</string>
+        <key>CFBundleURLSchemes</key>
+        <array>
+          <string>sensuss</string>
+        </array>
+        <key>CFBundleURLTypes</key>
+        <string>Viewer</string>
+      </dict>
+    </array>
+    <key>NSAppTransportSecurity</key>
+    <dict>
+      <key>NSExceptionDomains</key>
+      <dict>
+        <key>facebook.com</key>
+        <dict>
+          <key>NSIncludesSubdomains</key>
+          <true/>
+          <key>NSThirdPartyExceptionRequiresForwardSecrecy</key>
+          <false/>
+        </dict>
+        <key>fbcdn.net</key>
+        <dict>
+          <key>NSIncludesSubdomains</key>
+          <true/>
+          <key>NSThirdPartyExceptionRequiresForwardSecrecy</key>
+          <false/>
+        </dict>
+        <key>akamaihd.net</key>
+        <dict>
+          <key>NSIncludesSubdomains</key>
+          <true/>
+          <key>NSThirdPartyExceptionRequiresForwardSecrecy</key>
+          <false/>
+        </dict>
+      </dict>
+    </dict>
+    <key>LSApplicationQueriesSchemes</key>
```

Comment:
This and the following lines related to Facebook should be removed; otherwise, revert changes to this file (might need to edit in text editor rather than VS).